pkg/manager/manager.go

```diff
@@ -128,7 +132,7 @@ func New(config *rest.Config, options Options) (Manager, error) {
 	}
 
 	// Create the cache for the cached read client and registering informers
-	cache, err := options.newCache(config, cache.Options{Scheme: options.Scheme, Mapper: mapper, Resync: options.SyncPeriod})
+	cache, err := options.newCache(config, cache.Options{Scheme: options.Scheme, Mapper: mapper, Resync: options.SyncPeriod, Namespace: options.Namespace})
```

Comment:
When I need to watch all namespace events, set nil to `options.Namespace` or set other value?
Thanks.

Reply:
To watch all namespaces you set `options.Namespaces = ""` or [metav1.NamespaceAll](https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L278-L279) which is the same thing.
But since the default string value is empty you can just not set the Namespace option in `cache.Options` and it should watch all namespaces by default.